src/main/kotlin/org/rust/ide/formatter/RustAlignmentStrategy.kt

```diff
@@ -0,0 +1,42 @@
+package org.rust.ide.formatter
+
+import com.intellij.formatting.Alignment
+import com.intellij.lang.ASTNode
+
+/**
+ * Mimics [com.intellij.formatting.alignment.AlignmentStrategy], but offers more flexible API
+ */
+interface RustAlignmentStrategy {
```

Comment:
Actually I can translate the interface (only it) to Java and exploit https://kotlinlang.org/docs/reference/java-interop.html#sam-conversions as a kind of workaround for now.